aspnetcore/mvc/razor-pages/index.md

```diff
@@ -256,9 +256,9 @@ Add a *_ViewImports.cshtml* file:
 @addTagHelper "*, Microsoft.AspNetCore.Mvc.TagHelpers"
 ```
 
-The `@namespace` directive is a new feature that controls the namespace of the generated code allowing us to get rid of `@using` directives. If the `@namspace` directive is used directly in the page, the page will be put directly in the specified namespace. If the `@namespace` directive is inside a *_ViewImports.cshtml*, the specified namespace is only the prefix. The suffix is the dot-seperated relative path between the folder containing *_ViewImports.cshtml* and the folder containing the page.
+The `@namespace` directive is a new feature that controls the namespace of the generated code.  The `@namespace` directive replaces  `@using` directives. If the `@namspace` directive is used explicitly on a page, the page is in scope for the specified namespace. If the `@namespace` directive is contained in *_ViewImports.cshtml*, the specified namespace is only the prefix. The suffix is the dot-seperated relative path between the folder containing *_ViewImports.cshtml* and the folder containing the page.
```

Comment:
@halter73 I see. Can you suggest new wording?  

I think it's good info to say that it doesn't replace `@using`, and you'll frequently need both directives.